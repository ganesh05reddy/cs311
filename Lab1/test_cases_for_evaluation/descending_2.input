	.data
a:
	70
	80
	40
	20
	-10
n:
	5
