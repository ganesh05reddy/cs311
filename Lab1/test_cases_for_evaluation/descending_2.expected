Main Memory Contents:

0		: 80
1		: 70
2		: 40
3		: 20
4		: -10
